buildpacks/kotlin-gradle/bin/build

```diff
@@ -15,65 +24,74 @@ if [[ -r /etc/alpine-release ]]; then
     jdk_version="1.8.0_222"
 fi
 
-# Load user-provided build-time environment variables
-if compgen -G "$env_dir/*" > /dev/null; then
-  for var in "$env_dir"/*; do
-    declare "$(basename "$var")=$(<"$var")"
-  done
-fi
-
 echo "---> Installing JDK"
 
 # If it doesn't exist locally, create a JDK cache layer
```

Comment:
Do you want a period here too?

Reply:
Actually, it looks like we omit periods everywhere but below. /shrug

Any preference?

Reply:
Maybe just omit then